pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -768,7 +768,7 @@ def test_timestamp_multiindex_indexer():
         [
             pd.date_range(
                 start="2019-01-02T00:15:33",
-                end="2019-01-05T02:15:33",
+                end="2019-01-05T03:15:33",
```

Comment:
?

Reply:
This test was failing with my fix, but the previous behavior was wrong.

```
df.loc[pd.IndexSlice["2019-1-2":, "x", :], "foo"]
```

This statement should select every combination  beginning from 2019-01-02 to the end as long as the second component of MultiIndex is ``x`` (which holds true everywhere, because the second component is always ``x``). The test expected that every row would be selected except the last one. Which is pretty weird